test/cors.spec.js

```diff
@@ -0,0 +1,20 @@
+let helper = require('./helper');
+
+describe('CORS', function () {
+  it('GET /stories should allow cross origin', function(done) {
+    helper.request()
+          .get('/stories')
+          .end(function(err, res) {
+            res.should.to.have.header('Access-Control-Allow-Origin');
```

Comment:
In my opinion, this test only checks this resource actually supports CORS independently if you decide to allow some domains or not.

I have no problem in checking for the value too. Maybe at least I should check that the value is not empty.